x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/retry_delete_by_query_conflicts.ts

```diff
@@ -43,6 +49,16 @@ export async function retryIfDeleteByQueryConflicts<T>(
       await waitBeforeNextRetry(retryDelay);
       return await retryIfDeleteByQueryConflicts(logger, name, operation, retries - 1);
     }
+
+    failures.push(failure);
+  }
+
+  if (failures.length > 0) {
+    throw new Error(
+      `Unable to delete by query ${name} caused by ${failures
+        .map((failure) => failure.cause)
+        .join(', ')}`
+    );
```

Comment:
@maximpn Not sure I understand why we retry only when 409 conflicts occur instead of retrying on any error. Could we simplify it? Not only it would simplify the code of this test utility, it would also increase the resilience of tests.